assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/config.py

```diff
@@ -53,9 +53,3 @@ class SupportedTasks(_CustomEnum):
     IMAGE_CLASSIFICATION_MULTILABEL = "image-classification-multilabel"
     IMAGE_OBJECT_DETECTION = "image-object-detection"
     IMAGE_INSTANCE_SEGMENTATION = "image-instance-segmentation"
-
-
-class SupportedTextToImageModelFamily(_CustomEnum):
```

Comment:
@gauravrajguru - this was needed for optimisations, right?

Reply:
i shifted it to text_to_image configs as its needed in the exported mlflow wrapper